index.tt

```diff
@@ -6,152 +6,353 @@
 
 <link rel="alternate" type="application/rss+xml" title="RSS" href="/news-rss.xml" />
 
-<div class="nixos-org-index">
-  [%# Title for the site, used to keep the document tree rooted at h1. %]
-  <h1>NixOS</h1>
-  
-  <div class="jumbotron-two"><div>
-    <div class="intro-nix">
-      <h2>What is Nix?</h2>
-    
-      <p>
-      Nix is a powerful package manager for Linux and other Unix systems
-      that makes package management reliable and reproducible.  It
-      provides atomic upgrades and rollbacks, side-by-side installation
-      of multiple versions of a package, multi-user package management
-      and easy setup of build environments.
-      </p>
-    
-      <div class="learn-more">
-        <a class="btn btn-success" href="[%root%]nix/">
-          <i class="fa fa-chevron-right"></i> Learn more about Nix
-        </a>
+<div class="row-fluid">
+
+  <div class="span6">
+    <h1>Reproducible builds and deployments.</h1>
+    <p class="lead">
+      NixOS package manager can be used independently on any POSIX compliant
+      operating system to reliably share your development and build
+      environments across machines.
+    </p>
+    <p class="lead">
+      NixOS, an independently developed GNU/Linux distribution, aims to improve
```

Comment:
-> "NixOS is a Linux distribution that aims to...". "Independently" doesn't really mean anything (independent from what?).

Reply:
independent from debian or redhat. it's not based on any other distro

Reply:
To move on with this PR i will default to current text that we have on nixos.org. We can then fix it in a follow up PRs.